systers_portal/templates/meetup/base.html

```diff
@@ -0,0 +1,31 @@
+{% extends "base.html" %}
+{% load verbose_name %}
+
+{% block title %}
+  - {{ object.name }}
+{% endblock %}
+
+{% block content %}
+  <div class="mt40"></div>
+  <div class="row">
+    <div class="col-md-12">
+      <h1>{{ object.name }}</h1>
+      <hr/>
+    </div>
+    <div class="col-md-9">
+      <ul class="nav nav-pills">
+        <li> <a href="#">Upcoming</a> </li>
+        <li> <a href="#">Past</a></li>
+        <li> <a href="{% url 'display_meetup_location' slug 'organizers' %}">Organizers</a></li>
```

Comment:
Please don't paste false URLs, use `#` instead. It's easier to identify those empty URLs, then to scan all the URLs for false ones. There are other similar snippets throughout the code, please identify them and fix this.